src/test/java/org/sagebionetworks/bridge/services/AssessmentServiceTest.java

```diff
@@ -1136,9 +1166,7 @@ private void addMarkupToSensitiveFields(Assessment assessment) {
         assessment.setValidationStatus("some text</script>");
         assessment.setNormingStatus("Markup <object></object>can be <b>bold</b>.");        
         assessment.setTags(ImmutableSet.of("<scriopt>Мон ярсан суликадо</script>"));
-        Map<String, Set<String>> fields = new HashMap<>();
-        fields.put("<b>This</b>", ImmutableSet.of("<tag>Not right at all</tag>"));
-        assessment.setCustomizationFields(fields);
+        assessment.setCustomizationFields(CUSTOMIZATION_FIELDS);
```

Comment:
We should add markup to the customization fields to verify that they are being removed.

Reply:
Added